docs/openapi/components/schemas/common/IntentToImport.yml

```
@@ -1,10 +1,11 @@
 $linkedData:
-  term: IntentToSell
-  '@id': https://w3id.org/traceability#IntentToSell
-title: Intent To Sell
+  term: IntentToImport
+  '@id': https://w3id.org/traceability#IntentToImport
+title: Intent To Import
 description: >-
   A simple case where an entity (person or organization) wishes to notify some
-  third party that they intend to sell some product, likely into a different
+  third party that they intend to import some product, likely into a different
+  geographic region than it originated in.  This is a highly simplified version
```

```suggestion
  geographic region than that of its origin.  This is a highly simplified version
```
Or perhaps, `than where it originated.`